### ISSUE TITLE
Milestone 4

### ISSUE THREAD

**Issue by LizIsTired** [LizIsTired/StarminerRecrafted#5]
- [ ] Refactor code

This is important to be separate so I can test each change to make sure it all works. Will likely include moving things to other files, such as helper files to get rid of duplicated code.

Will bump version to 0.4.0. Could potentially bump to 1.0.0 if no issues are present.

**Comment by Raylo555** [comment]
Are you still working on it?

**Comment by LizIsTired** [comment]
Nope! Other people are working on it on this server: https://discord.gg/k8q7WGAavB
